release notes/v0.48.0.md

```diff
@@ -0,0 +1,257 @@
+
+```js
+...
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    page.throttleCPU({ rate: 4 });
+...
+```
+
+### Add support for browser module's `page.throttleNetwork` [browser#1094](https://github.com/grafana/xk6-browser/pull/1094)
+
+Allows users to throttle the characteristics of the network from chrome/chromium's perspective, which helps emulate poor network connections when testing the website's frontend. The mandatory argument is of type `NetworkProfile` and is defined as:
+
+```ts
+export interface NetworkProfile {
+    /*
+     * Minimum latency from request sent to response headers received (ms).
+     */
+    latency: number;
+    
+    /*
+     * Maximal aggregated download throughput (bytes/sec). -1 disables download
+     * throttling.
+     */
+    download: number;
+    
+    /*
+     * Maximal aggregated upload throughput (bytes/sec). -1 disables upload
+     * throttling.
+     */
+    upload: number;
+}
+```
+
+You can either define your own network profiles or use the ones we have defined by importing `networkProfiles` from the `browser` module. The documentation for this API can be found [here](https://grafana.com/docs/k6/v0.48.x/javascript-api/k6-experimental/browser/page/throttlenetwork).
+
+```js
+import { browser, networkProfiles } from 'k6/experimental/browser';
+...
+  const context = browser.newContext();
+  const page = context.newPage();
+
+  try {
+    page.throttleNetwork(networkProfiles['Slow 3G']);
+...
+```
+
+## UX improvements and enhancements
+
+- [browser#1074](https://github.com/grafana/xk6-browser/pull/1074) Adds a new `browser.closeContext()` [method](https://k6.io/docs/javascript-api/k6-experimental/browser/closecontext/) to facilitate closing the current active browser context.
+- [#3370](https://github.com/grafana/k6/pull/3370) The k6 got a new flag `--profiling-enabled` which enables exposing [pprof](https://go.dev/blog/pprof) profiling endpoints. The profiling endpoints are exposed on the same port as the HTTP REST API under the `/debug/pprof/` path. This could be useful for the extension developers.
+- [#3442](https://github.com/grafana/k6/pull/3442) The k6 now supports a new `--version` flag. It has the same output as `k6 version` command. Thanks, @ffapitalle!
+- [#3423](https://github.com/grafana/k6/pull/3423) The InfluxDB output got an environment variable `K6_INFLUXDB_PROXY` which allows specifying proxy. Thanks, @IvanovOleg!
+- [#3398](https://github.com/grafana/k6/pull/3398) The k6 cloud traces are enabled by default.
+- [#3400](https://github.com/grafana/k6/pull/3400) The binary-based cloud output (a.k.a. cloud output v2) is now the default version when the test streams metrics from a local test run via `-o cloud`.
+- [#3452](https://github.com/grafana/k6/pull/3452). Added `fsext.Abs` helper function.
+- [#3446](https://github.com/grafana/k6/pull/3446) A log tokenizer moved to `lib/strvals` package.
```

Comment:
Should this be just:
```suggestion
- [#3446](https://github.com/grafana/k6/pull/3446) Log tokenizer moved to `lib/strvals` package.
```
Also, should this be placed under _"Maintenance and internal improvements"_? As it is just a refactor/renamig plus moving to another package?

Reply:
:shrug: I've put this here since now it's publicly exposed API, but generally fine with moving. Should I?

Reply:
Okay, will move then :+1: